app/models/user.rb

```diff
@@ -1,9 +1,15 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] Currently, a user can not sign up, log in or confirm their details when they fill in the form inputs. This is what happens when I try to sign up, log in or confirm email. Kindly set up `devise` properly. You can read this [article](https://riptutorial.com/devise/example/12284/installation-or-setup) as a guide.

![Rollback](https://user-images.githubusercontent.com/77106962/162334310-208f76e9-df96-42e3-8a83-50f5efbeb7b1.png)

![401-page](https://user-images.githubusercontent.com/77106962/162334644-c3f6a280-7290-4c06-9f9c-1fb9f098c53f.png)

Reply:
@Gambit142

Reply:
@Gambit142  thank you for the review, I Appreciate the help